src/UhdmAst.cpp

```diff
@@ -1847,22 +1892,21 @@ AstNode* visit_object(vpiHandle obj_h, UhdmShared& shared) {
     }
     case vpiPackage: {
         AstPackage* package = nullptr;
-        auto it = shared.package_map.find(objectName); // In case it has been created because of an import
-        if (it != shared.package_map.end()) package = it->second;
-        else package = new AstPackage(new FileLine(objectName), objectName);
+        auto it = shared.package_map.find(
+            objectName);  // In case it has been created because of an import
+        if (it != shared.package_map.end())
+            package = it->second;
+        else
+            package = new AstPackage(new FileLine(objectName), objectName);
```

Comment:
If you're changing formatting, update `package` to `packagep` as well.

Reply:
Done. I changed formatting with clang-format, so I didn't notice that the name is against the convention.